atlas_densities/densities/inhibitory_neuron_densities_optimization.py

```diff
@@ -491,11 +492,30 @@ def _check_variables_consistency(
             if np.isfinite(deltas.loc[region_name, cell_type]):
                 for desc_id in id_set:
                     if np.isnan(x_result.loc[desc_id, cell_type]):
-                        raise AtlasDensitiesError(
-                            f"Cell count estimate of region named '{region_name}' for cell type "
-                            f"{cell_type} was given for certain whereas the cell count of "
-                            f"descendant id {desc_id} is not certain."
-                        )
+                        if ((deltas.loc[region_name, cell_type] in [0.0, np.inf]) & 
+                           (x_result.loc[id_, cell_type] == 0.0) or 
+                           (np.isnan(x_result.loc[id_, cell_type]))):
+                            ''' 
```

Comment:
this should be a comment, not an inline string

Reply:
ok